<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tablebquery</class>
 <widget class="QDialog" name="tablebquery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>关键字</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>升序</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="downButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>降序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>排序方式</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ridButton_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>房间号</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cidButton_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>顾客id</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cnameButton_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>顾客名</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="startTButton_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>开始时间</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="endTButton_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>结束时间</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>296</width>
     <height>236</height>
    </rect>
   </property>
   <property name="firstDayOfWeek">
    <enum>Qt::Sunday</enum>
   </property>
   <property name="verticalHeaderFormat">
    <enum>QCalendarWidget::NoVerticalHeader</enum>
   </property>
   <property name="navigationBarVisible">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
